tests/e2e/api/src/models/orders/shared/types.ts

```diff
@@ -0,0 +1,66 @@
+/**
+ * An order's status.
+ *
+ * @typedef OrderStatus
+ */
+export type OrderStatus = 'pending' | 'processing' | 'complete' | 'on-hold' | 'refunded'
+	| 'cancelled' | 'failed' | 'trash' | string;
```

Comment:
Unclear why this type ends in `| string` while the others below don't.

Reply:
It allows for custom statuses.